addons/mail/static/src/models/emoji_grid_view.js

```diff
@@ -16,12 +16,28 @@ registerModel({
             if (!this.emojiPickerViewOwner.emojiCategoryBarView.activeCategoryView) {
                 return clear();
             }
+            if (this.emojiPickerViewOwner.emojiSearchBar.currentSearch !== "") {
+                const filtered_emojis = this.emojiPickerViewOwner.emojiCategoryBarView.activeCategoryView.emojiCategory.allEmojis.filter(this._filterEmoji);
```

Comment:
This `filteredEmojis` is also theoretically inefficient.
If not perceivable performance issues, it's ok.
Making a more efficient code here requires some effort, so can be postponed